docs/topics/spiders.rst

```diff
@@ -42,6 +42,19 @@ Even though this cycle applies (more or less) to any kind of spider, there are
 different kinds of default spiders bundled into Scrapy for different purposes.
 We will talk about those types here.
 
+.. note:: Spider callbacks can be implemented as ``generators``. In Python 3.3+
```

Comment:
I don’t think we need to document any of this, at least not in this page.

We could add an entry to the FAQ if we though some people may need additional information on top of the warning. However, in this case, I think the warning message is quite detailed, and already points to Python documentation about the topic.

Reply:
Fair enough. Although it took me a while to write this and be happy about the wording :cry: